foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -334,6 +334,14 @@ def requeue_downloader_job(last_job: DownloaderJob) -> None:
         last_job.save()
         return
 
+    first_sample = original_file.samples.first()
+    if first_sample and first_sample.is_blacklisted:
+        last_job.no_retry = True
+        last_job.failure_reason = "Sample run accession has been blacklisted by SRA."
+        last_job.save()
+        logger.info("Avoiding requeuing for DownloaderJob for blacklisted run accession: " + str(first_sample.accession_code))
```

Comment:
This might also be rather chatty because presumably we have downloader jobs out there already for all the blacklisted accessions.

Reply:
I'd like to keep this for now to make sure it's working as intended.